products/idn/tools/sdk/powershell.md

```diff
@@ -0,0 +1,352 @@
+
+If you already have a version of the PowerShell SDK installed, you can install a new version side-by-side with it by adding the `-Force` parameter to the end of your `Install-Module` commmand. 
+
+### Configure the SDK
+To configure the SDK, create a configuration file or save your configuration as environment variables. 
+You can use any of the following ways to do so: 
+
+#### Manual Configuration
+
+One way to create a configuration file is to create a "config.yaml" file in your project and specify the following information in it: 
+
+```yaml
+activeenvironment: example # the key that identifies the currently active environment
+authtype: pat # currently only pat and pipeline are supported if the ENV VAR SAIL_AUTH_TYPE is configured to "pipeline" it will override this value
+customexporttemplatespath: "" # the path to the users custom export templates file if one is provided
+customsearchtemplatespath: "" # the path to the users custom search templates file if one is provided
+debug: false # the debug setting  
+environments: # the configured environments  
+  example:
+    baseurl: https://example.api.identitynow.com
+    pat:
+      accesstoken: example-access-token
+      clientid: example-client-id
+      clientsecret: example-client-secret
+      expiry: example-access-token-expiry
+    tenanturl: https://example.identitynow.com
+```
+
+You must specify the following information: 
+- `activeenvironment`: This key identifies the current active environment the SDK is connecting to. This environment name refers to your IDN tenant name. In the example, the key is "example". You must also make sure the environment name listed under `environments` matches the `activeenvironment`. 
+- `authtype`: The authentication type. Currently only "pat" and "pipeline" are supported. Configuring ENV VAR SAIL_AUTH_TYPE to "pipeline" overrides this value. In the example, the authentication type is "pat". You must also make sure the authentication type listed under the environment name "example" matches the `authtype`. 
+- `baseurl` and `tenanturl`: These refer to your IDN tenant URL. 
+- `clientsecret`: The PAT's client secret. 
+- `clientid`: The PAT's client ID. 
+
+Here's an example: 
+
+```yaml
+activeenvironment: devrel # the key that identifies the currently active environment
+authtype: pat # currently only pat and pipeline are supported if the ENV VAR SAIL_AUTH_TYPE is configured to "pipeline" it will override this value
+customexporttemplatespath: "" # the path to the users custom export templates file if one is provided
+customsearchtemplatespath: "" # the path to the users custom search templates file if one is provided
+debug: false # the debug setting  
+environments: # the configured environments 
+  example:
+    baseurl: https://devrel.api.identitynow.com
+    pat:
+      accesstoken: example-access-token
+      clientid: g0567b766b413b22c05c66e75d532f1b
+      clientsecret: cabd0e950a7230b63c1ff45be33fb22065b382b6251a73c61177a8bb5482fcc7
```

Comment:
Hey @james-haytko-sp, are these your credentials to access the devrel tenant? If so, we will want to remove these from this PR and use dummy values.

Reply:
Actually, these are dummy values, so no worries there.